airflow/contrib/hooks/gcp_function_hook.py

```diff
@@ -46,11 +46,25 @@ def __init__(self,
         super(GcfHook, self).__init__(gcp_conn_id, delegate_to)
         self.api_version = api_version
 
+    @staticmethod
+    def _full_location(project_id, location):
+        """
+        Retrieve full location of the function in the form of
+        projects/<GCP_PROJECT_ID>/locations/<GCP_LOCATION>
+        :param project_id: The Google Cloud Project project_id where the function belongs.
```

Comment:
Empty line needed above this

Reply:
I wish flake complained about it before :). Will do.